pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/HealthCheck.kt

```diff
@@ -0,0 +1,142 @@
+package gov.cdc.ocio.processingstatusapi
+
+import com.azure.cosmos.models.CosmosQueryRequestOptions
+import com.azure.messaging.servicebus.ServiceBusClientBuilder
+import com.azure.messaging.servicebus.models.ServiceBusReceiveMode
+import com.microsoft.azure.servicebus.primitives.ServiceBusException
+import gov.cdc.ocio.processingstatusapi.cosmos.CosmosRepository
+import gov.cdc.ocio.processingstatusapi.models.Report
+import gov.cdc.ocio.processingstatusapi.plugins.AzureServiceBusConfiguration
+import mu.KotlinLogging
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+import kotlin.system.measureTimeMillis
+
+
+abstract class HealthCheckSystem {
+
+    var status: String = "DOWN"
+
+    var healthIssues: String? = ""
+
+    open val service: String = ""
+}
+
+class HealthCheckCosmosDb: HealthCheckSystem() {
+    override val service = "Cosmos DB"
+}
+
+class HealthCheckServiceBus: HealthCheckSystem() {
+    override val service: String = "Azure Service Bus"
+}
+
+/**
+ * Run health checks for the service.
+ *
+ * @property status String?
+ * @property totalChecksDuration String?
+ * @property dependencyHealthChecks MutableList<HealthCheckSystem>
+ */
+class HealthCheck {
+
+    var status : String? = "DOWN"
```

Comment:
If status has default value of DOWN will it ever be null

Reply:
Removed nullable '?' from status signature since can never be null